README.md

```diff
@@ -99,6 +103,9 @@ Default value is `false`
 `eagerInject` - whether to process `eagerInject` fields in DI resolver configuration, which instantiates and caches module immediately. Disabling this will make app startup slightly faster.
 Default value is `false`
 
+`strictBooleanEnforced` - whether to throw an error if `enabled` field in a resolver configuration is set with unsupported value (anything different from `true` and `false`). It is recommended to set this to `true`, which will be a default value in the next semver major release.
+Default value is `false`
```

Comment:
I was thinking to set it to true already, but let's sync with the wrapped module tho

https://www.npmjs.com/package/awilix-manager

Reply:
@Eomm I'm OK to set it to true and release a new semver major.

Reply:
although we can do it in a follow-up PR for backwards compatibility

Reply:
No, it is not mentioned as security warning - so it is good this default 👍🏼